TestModels/Errors/src/SimpleErrorsImpl.dfy

```diff
@@ -54,7 +54,7 @@ module SimpleErrorsImpl refines AbstractSimpleErrorsOperations  {
       // TODO: Rewrite this as an actual extern.
       var opaqueObject := new SomeOpaqueGeneratedTypeForTesting();
 
-      var res := Error.Opaque( obj := opaqueObject );
+      var res := Error.Opaque( obj := opaqueObject, alt_text := "Some Generated Test" );
```

Comment:
I'm still not a fan of having the Dafny type use snakecase, but I think this is a 2-way door.

The Dafny-generated type name is internal-only. I believe that if we were to camelcase this member name in the future, it's only a breaking change to internal interfaces, and customers won't need to update their code.